addons/mail/static/src/core/common/settings_model.js

```diff
@@ -38,6 +39,53 @@ export class Settings extends Record {
      * Normalized [0, 1] volume at which the voice activation system must consider the user as "talking".
      */
     voiceActivationThreshold = 0.05;
+    /**
+     * General notification settings for channels
+     * @type {'mentions'|'all'|'no_notif'}
+     */
+    channel_notifications = Record.attr("mentions", {
+        compute() {
+            return this.channel_notifications === false ? "mentions" : this.channel_notifications;
+        },
+    });
+    selected_mute_duration = Record.attr(false, {
```

Comment:
this means which "duration button" is selected so that the chocie is stored locally and when refresh the page it is still there. it is different from the value in the server.

Reply:
I don't understand the point of saving a local value of something that we store on the server.

Reply:
It should be there on refresh because the server will give the correct value.